### ISSUE TITLE
Добавить оставшиеся страны

### ISSUE THREAD

**Issue by grandsbor** [OpenCorpora/opencorpora#86]
Бонэйр; Синт-Эстатиус; Саба; Бруней-Даруссалам; Гвинея-Бисау; Дуннин; Кабо-Верде; Кирибати; Коморы; Кюрасао; Майотта; Ниуэ; Палау; Питкерн; Сен-Бартелеми; Сен-Мартен; Сен-Пьер; Микелон; Синт-Мартен; Синт-Эстатиус; Сомалиленд; Тимор-Лесте; Транскей; Уоллис; Футуна; Борнео
(отсюда: https://code.google.com/p/opencorpora/issues/detail?id=432)

**Comment by svbichineva** [comment]
Миша, ты можешь?

**Comment by kmike** [comment]
Проверил - вот все символы, которые встречались в словаре (по порядку): 
`'-абвгдежзийклмнопрстуфхцчшщъыьэюяё’`

Латинских букв нет, зато есть два разных апострофа.

С `'` есть одно слово - `о'кей`.
C `’` тоже одна лексема - `кот-д’ивуар` и его формы.

Может, какой-нибудь один использовать?

**Comment by grandsbor** [comment]
Да, давайте выберем один, а всё остальное, кроме букв, дефиса и этого одного запретим в коде

**Comment by svbichineva** [comment]
Дим, выбери на свой вкус и закрой задачу про страны, пожалуйста
